### ISSUE TITLE
bug: Aurora Postgres missing log types error

### ISSUE THREAD

**Issue by vladshcherbin** [cdklabs/cdk-nag#759]
### What is the problem?

Using Aurora, Postgres and `cloudwatchLogsExports` set to `['postgresql']` results in errors: 

```
HIPAA.Security-RDSLoggingEnabled[LogExport::postgresql]: The RDS DB instance does not have all CloudWatch log types exported - (Control IDs: 164.308(a)(3)(ii)(A), 164.308(a)(5)(ii)(C)). To help with logging and monitoring within your environment, ensure Amazon Relational Database Service (Amazon RDS) logging is enabled. With Amazon RDS logging, you can capture events such as connections, disconnections, queries, or tables queried.This is a granular rule that returns individual findings that can be suppressed with 'appliesTo'. The findings are in the format 'LogExport::<log>' for exported logs. Example: appliesTo: ['LogExport::audit'].

HIPAA.Security-RDSLoggingEnabled[LogExport::upgrade]: The RDS DB instance does not have all CloudWatch log types exported - (Control IDs: 164.308(a)(3)(ii)(A), 164.308(a)(5)(ii)(C)). To help with logging and monitoring within your environment, ensure Amazon Relational Database Service (Amazon RDS) logging is enabled. With Amazon RDS logging, you can capture events such as connections, disconnections, queries, or tables queried.This is a granular rule that returns individual findings that can be suppressed with 'appliesTo'. The findings are in the format 'LogExport::<log>' for exported logs. Example: appliesTo: ['LogExport::audit'].
```

Trying to use `['postgresql', 'upgrade']` results in cdk error: 

```
Error: Unsupported logs for the current engine type: upgrade
````

[CloudFormation docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-enablecloudwatchlogsexports) shows `postgresql` is the only valid type.

### Reproduction Steps

Create an Aurora db cluster and run cdk synth with cdk nag:

```ts
new DatabaseCluster(this, 'cluster', {
  engine: DatabaseClusterEngine.auroraPostgres({ version: AuroraPostgresEngineVersion.VER_13_4 }),
  cloudwatchLogsExports: ['postgresql'],
  // rest props
})
```

### What did you expect to happen?

No error when `cloudwatchLogsExports` is set to `['postgresql']`.

### What actually happened?

A missing log types error.

### cdk-nag version

2.12.13

### Language

Typescript

**Comment by skinny85** [comment]
Hi @vladshcherbin,

thanks for opening the issue.

Can you explain where these errors come from?

```
HIPAA.Security-RDSLoggingEnabled[LogExport::postgresql]:
HIPAA.Security-RDSLoggingEnabled[LogExport::upgrade]:
```

These are not CloudFormation errors, right?

Also, if the CloudFormation docs say `'postgres'` is the only supported log type, then it seems to me the CDK validation is correct?

Thanks,
Adam

**Comment by skinny85** [comment]
Nvm, for some reason I thought this was the CDK repo 😛. Please ignore me!

**Comment by dontirun** [comment]
Thanks for reporting! 

It looks like any `Aurora` instances have their logs managed on a cluster level. So we need to

1. Ignore any `Aurora` based instances
2. Include `Aurora` Clusters
   - Update proper log types for both  `Aurora MySQL`  and  `Aurora PostgreSQL`

**Comment by dontirun** [comment]
Fixed in versions `1.12.14` and `2.12.14`

**Comment by vladshcherbin** [comment]
Wow, this was quick. Thank you! 🥳
